bin/templates/regress_rmdb.j2

```diff
@@ -0,0 +1,94 @@
+{% import 'regress_macros.j2' as regress_macros -%}
```

Comment:
This looks like a tool-generated file, is that correct?  Can we place an SPDX header on this file?

Reply:
I don't totally understand your question. Are you talking about the whole *_rmdb.j2 file ? of only the _macros.j2 file and the import line ? 

If it's for the whole rmdb file, it's not, I wrote this by hand to manage regressions with Siemens tools, but yes I forgot to add a header on this one. 

If you're talking about the import line, it was my first time using Jinja2 templating, and I simply get inspired (a lot) by the already existing regress_vsif.j2 file, so I simply copied the imported line, and so the file, which is also missing a header as I can see.

Reply:
Thanks.  I was asking about whether the file was generated or not because we have had problems in the past with EDA tools generating files based on closed source artifacts.   If you have generated this manually, we should be fine.

Please do add a license header.  A two-liner will suffice:
```
// Copyright 2023 Dolphin Design
// SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
```

Reply:
```suggestion
// Copyright 2023 Dolphin Design
// SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1

{% import 'regress_macros.j2' as regress_macros -%}
```